sources/packages/web/src/components/common/AwardTable.vue

```diff
@@ -47,17 +48,22 @@ export default defineComponent({
         (award) => award.offeringIntensity === props.offeringIntensity,
       ),
     );
+    const { getFormattedMoneyValue } = useFormatters();
 
     const getAwardValue = (awardType: string): string | number | Date => {
-      const awardValue =
-        props.awardDetails[`${props.identifier}${awardType.toLowerCase()}`];
+      const awardValue = props.awardDetails[
+        `${props.identifier}${awardType.toLowerCase()}`
+      ] as number;
       // If the award is defined but no values are present it means that a receipt value is missing.
       if (awardValue === null) {
         return "-";
       }
       // If the award in not defined at all it means that the award is not eligible and it was not
       // part of the disbursement calculations output.
-      return awardValue ?? "(Not eligible)";
+      if (awardValue === undefined) {
+        return "(Not eligible)";
+      }
+      return getFormattedMoneyValue(awardValue);
```

Comment:
`return awardValue ? getFormattedMoneyValue(awardValue) : "(Not eligible)";`

Reply:
It is the opposite @andrepestana-aot when undefined `"(Not eligible)"` needs to return. Are you saying this as a suggestion?